src/project-tests/tribute-page-tests.js

```diff
@@ -3,6 +3,13 @@ import $ from 'jquery';
 import { responsiveWebDesignStack } from '../utils/shared-test-strings';
 
 export default function createTributePageTests() {
+  // Save the HTML element type of the element with id="main"
+  // Needed for checking if we should run the tests for old or new projects
+  let mainElement = null;
+  const hasMain = document.getElementById('main');
+  if (hasMain) {
+    mainElement = hasMain.localName;
```

Comment:
This was a good solution, but there may not be a problem with the test if you keep the `id="img-div"` regardless of tag. We should keep it simple. I left a question in the review...